staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -37,7 +37,7 @@ type Versioner interface {
 	// UpdateList sets the resource version into an API list object. Returns an error if the object
 	// cannot be updated correctly. May return nil if the requested object does not need metadata
 	// from database.
-	UpdateList(obj runtime.Object, resourceVersion uint64) error
+	UpdateList(obj runtime.Object, resourceVersion uint64, next string) error
```

Comment:
we need to either:

1. document next is optional
2. or make next an option arg by doing (obj runtime.Object, resourceVersion uint64, opts listopts...)